rfcs/text/0000-analytics-instrumentation-plan.md

```diff
@@ -0,0 +1,137 @@
+### DCP PR:
+
+# Analytics instrumentation plan
+
+## Summary
+
+To support collection of the Human Cell Altas Data Coordination Platform (HCA DCP) user metrics desired by the DCP
+development team, the HCA DCP service must be instrumented with a user tracking solution. This document describes the
+goals for the tracking solution and a proposal for instrumenting HCA DCP service components.
+
+## Author(s)
+
+* [Matt Weiden](mailto:mweiden@chanzuckerberg.com)
+* [Marcio Von Muhlen](mailto:mvonmuhlen@chanzuckerberg.com)
+* [Adrienne Sussman](mailto:%20asussman@chanzuckerberg.com)
+
+## Shepherd
+
+## Motivation
+
+The development team needs quantitative data on application usage to drive informed decisions on how HCA DCP should
+evolve as a product. The following goals address this need.
+
+* Respect user privacy: do not gather personally identifiable information nor build user profiles
+* Gather anonymized user browsing activity in the data portal and data browser
+* Gather anonymized data download events initiated from any user interface (web, CLI, API)
+* Gather anonymized data upload events
+* Enable restricted access control to the user metrics data set
+* Enable non-developers to access and analyze the resulting data set to improve the HCA DCP inform the HCA scientific
+  leadership team
+
+### User Stories or Use Cases
+
+* As an HCA PM, I want to analyze quantitative metrics of user actions on DCP, to make informed decisions on product
```

Comment:
HCA PM is very internal jargon, it needs to be defined somewhere or made clearer